src/binheap.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding utf-8 -*-
+
+
+class Binheap(object):
+
+    def __init__(self, iterable=None):
+        '''Initialise a heap from an iterable or an empty heap'''
+        if iterable is None:
+            self.heap = None
```

Comment:
This is a Bad Idea. Even though Python is a loosely typed language, that doesn't mean you should necessarily allow a class attribute to change types (between `None` type and some iterable type). Better to remain consistent.